configure.in: fix linux/errqueue.h not found

The do_compile failed:
...
|tmp/work/core2-64-poky-linux/socat/1.7.2.4-r0/socat-1.7.2.4/xio-ip.c:
In function 'xiolog_ancillary_ip':
|tmp/work/core2-64-poky-linux/socat/1.7.2.4-r0/socat-1.7.2.4/xio-ip.c:520:12:
error: dereferencing pointer to incomplete type
|         err->ee_errno, '\0', err->ee_origin, '\0', err->ee_type, '\0',
|            ^
| ake: *** [xio-ip.o] Error 1
...

For 1.7.2.4, the header check of 'linux/errqueue.h ' was not correctly,
but the old version 1.7.2.3 worked, so we reference 1.7.2.3 to fix this
issue.

Upstream-Status: Pending
Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 configure.in | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/configure.in b/configure.in
index 70922f9..bdfbe98 100644
--- a/configure.in
+++ b/configure.in
@@ -79,8 +79,7 @@ AC_HEADER_RESOLV()
 
 AC_CHECK_HEADERS(termios.h linux/if_tun.h)
 AC_CHECK_HEADERS(net/if_dl.h)
-AC_CHECK_HEADERS(linux/types.h)
-AC_CHECK_HEADER(linux/errqueue.h, AC_DEFINE(HAVE_LINUX_ERRQUEUE_H), [], [#include <linux/types.h>])
+AC_CHECK_HEADERS(linux/types.h linux/errqueue.h)
 AC_CHECK_HEADERS(sys/utsname.h sys/select.h sys/file.h)
 AC_CHECK_HEADERS(util.h bsd/libutil.h libutil.h sys/stropts.h regex.h)
 AC_CHECK_HEADERS(linux/fs.h linux/ext2_fs.h)
-- 
1.8.1.2

